packages/cashier/src/stores/payment-agent-store.ts

```diff
@@ -240,6 +239,12 @@ export default class PaymentAgentStore {
 
         this.setIsSearchLoading(false);
 
+        this.routeToCashierDepositIfAgentVisible();
```

Comment:
what's the reason behind this change?

Reply:
This is to reduce the congnitive complexity of the function, I separated the logic in a different function

Reply:
got it thanks